### PR TITLE
[SAIC-438] Templating of creating a database connection resource

### PR DESCRIPTION
**Pull request by mirrorcoder**

### CODE REVIEW

File: cloudferrylib/os/network/neutron.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def __init__(self, config, cloud):
         self.filter_tenant_id = None
         self.ext_net_map = \
             utl.read_yaml_file(self.config.migrate.ext_net_map) or {}
+        self.mysql_connector = cloud.mysql_connector('neutron')
```
Comment (roman-verchikov) [inline cloudferrylib/os/network/neutron.py]: This may fail for pre-havana releases, where neutron was called "quantum".

**Comment by roman-verchikov** [comment]
looks good other than one inline, please fix

**Comment by mirrorcoder** [comment]
I changed logic for mysql config in configuration.ini, base config is section src_mysql/dst_mysql another sections will override mysql credentional

File: cloudferrylib/os/network/neutron.py (hunk 1 above)
Comment (mirrorcoder) [inline cloudferrylib/os/network/neutron.py]: if we need another db_name for network then we can change in configuration.ini

**Comment by roman-verchikov** [comment]
please rebase

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**